internal/merkle/inmemory/tree_test.go

```diff
@@ -124,16 +124,16 @@ var testProofs = []proofTestVector{
 	}},
 }
 
-func decodeHexStringOrPanic(hs string) []byte {
+// hx decodes a hex string or panics.
+func hx(hs string) []byte {
```

Comment:
This change is unrelated to this CL, right?
Although I like the shortness, after this PR, it is unclear from looking at the name of this function wether is decodes or encodes a string in hex

Reply:
Yeah, this could be separated, I just did it as a drive-by because otherwise the lists of constants would look too verbose.

This file was old and ugly anyway, so #2707 refactors it. After the refactoring, `hx` is used only with constants that are inlined, e.g. `hx("6e340b9cffb37a989ca544e6bb780a2c78901d3fb33738768511a30617afa01d")`, and never used with variables like `hx(someVariable)`. So it's obvious from context that it decodes, and the pros is that the name is short.